defaults/initrd.scripts

```diff
@@ -1059,8 +1059,7 @@ run_shell() {
 }
 
 fs_type_in_use() {
-	fs_type=$1
-	cut -d ' ' -f 3 < /proc/mounts | grep -Fq "${fs_type}"
+	grep -Eqse "^[^ ]+ [^ ]+ ${1} " /proc/mounts
```

Comment:
please see my comments on PR#27 and switch to using awk instead.

Much cleaner to understand.